File: tests/fixtures/php_project/src/Calculator.php
<?php

namespace Example;

class Calculator
{
    public static function add(int $a, int $b): int
    {
        return $a + $b;
    }

    public static function multiply(int $a, int $b): int
    {
        return $a * $b;
    }
}

// For command line execution
if ($argv[0] === __FILE__) {
    echo "Hello from PHP!\n";
    echo "5 + 3 = " . Calculator::add(5, 3) . "\n";
    echo "5 * 3 = " . Calculator::multiply(5, 3) . "\n";
}

File: tests/fixtures/ruby_project/lib/calculator.rb
module Calculator
  def self.add(a, b)
    a + b
  end

  def self.multiply(a, b)
    a * b
  end
end

if __FILE__ == $0
  puts "Hello from Ruby!"
  puts "5 + 3 = #{Calculator.add(5, 3)}"
  puts "5 * 3 = #{Calculator.multiply(5, 3)}"
end
